core-services/release-controller/_releases/release-ocp-4.14-multi.json

```diff
@@ -45,6 +45,13 @@
       "prowJob": {"name": "periodic-ci-openshift-multiarch-master-nightly-4.14-upgrade-from-stable-4.13-ocp-e2e-aws-ovn-heterogeneous-upgrade"},
       "upgrade": true,
       "upgradeFrom": "PreviousMinor"
+    },
+    "multi-azure-ovn-upgrade": {
+      "disabled": true,
```

Comment:
The release-controller will completely ignore jobs that have `"disabled": true`.  Why bother even adding the job?

Reply:
oops..didn't mean to do that. lifted off the priv files